src/interfaces/filing-i.ts

```diff
@@ -0,0 +1,162 @@
+import { AmalgamationTypes, CorpTypeCd, FilingTypes } from '@bcrs-shared-components/enums'
+import type {
+  ApiDateTimeUtc,
+  CommentIF,
+  FormattedDateTimeGmt,
+  IsoDatePacific,
+  SpecialResolutionIF
+} from '@bcrs-shared-components/interfaces'
+
+import type { DocumentI, FilingStatusE, FilingSubTypeE } from '#imports'
+import type { EffectOfOrderTypeE } from '~/enums/effect-order-type-e'
+
+/**
+ * A filing object from the Legal API ("filings" call). This is the newer response with extra metadata
+ * instead of separate business / documents / header / filing objects.
+ * This object is also what the Filings History List uses directly.
+ */
+// todo: fixme: maybe create additional interface that will have GMT times converted into ISO UTC datetime
+// see effectiveDate and submittedDate bellow
+// or, even better create/update existing endpoint that would return ISO UTC dates
+export interface ApiResponseFilingI {
```

Comment:
There is a `StateFilingHeaderI` interface. Maybe we can combine them into the same file?